File: Nível 1/2417 - Campeonato/main.lua
infos = io.read()
vc, ec, gc, vf, ef, gf = infos:match("(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)")
vc, ec, gc, vf, ef, gf = tonumber(vc), tonumber(ec), tonumber(gc), tonumber(vf), tonumber(ef), tonumber(gf)

pontosCor = (vc * 3) + (ec * 1)
pontosFla = (vf * 3) + (ef * 1)

if pontosCor > pontosFla then
    print("C")
elseif pontosFla > pontosCor then
    print("F")
elseif pontosFla == pontosCor then
    if gc > gf then
        print("C")
    elseif gf > gc then
        print("F")
    elseif gc == gf then
        print("=")
    else
        print("erro")
    end
end

File: Nível 1/2417 - Campeonato/main.py
infos = input().split(" ")
vc, ec, gc = int(infos[0]), int(infos[1]), int(infos[2])
vf, ef, gf = int(infos[3]), int(infos[4]), int(infos[5])

pontosCor = (vc * 3) + (ec * 1)
pontosFla = (vf * 3) + (ef * 1)

if pontosCor > pontosFla:
    print("C")
elif pontosFla > pontosCor:
    print("F")
elif pontosFla == pontosCor:
    if gc > gf:
        print("C")
    elif gf > gc:
        print("F")
    elif gc == gf:
        print("=")
    else:
        print("erro")